sources/web/datalab/static.ts

```diff
@@ -112,6 +116,23 @@ function datalabFileExists(filePath: string) {
 }
 
 /**
+ * Sends a static 'custom.css' file located within the user's config directory.
+ *
+ * @param userId the ID of the current user.
+ * @param response the out-going response associated with the current HTTP request.
+ */
+function sendUserCustomTheme(userId: string, response: http.ServerResponse): void {
+    var customThemePath = path.join(settings.getUserConfigDir(userId), CUSTOM_THEME_FILE);
+    fs.stat(customThemePath, function(e, stats) {
```

Comment:
We can probably cache the state (whether custom theme file available) so we don't need to read from disk for each request. In the future if they change theme from UI, we can trigger the state refreshing (assume the UI change will come soon).